types/react-router/index.d.ts

```diff
@@ -122,7 +122,7 @@ export interface StaticRouterProps {
 
 export class StaticRouter extends React.Component<StaticRouterProps, any> {}
 export interface SwitchProps {
-    children?: React.ReactNode | undefined;
+    children?: (Route | Redirect)[] | undefined;
```

Comment:
This is not the same as `<Route />` which is what's documented.

Reply:
Which Route should I use instead?

Reply:
You probably want to use an element created from `Route` not the component itself.